Core/include/Acts/EventData/TrackStateProxy.hpp

```diff
@@ -776,6 +817,29 @@ class TrackStateProxy {
     m_traj->allocateCalibrated(m_istate, measdim);
   }
 
+  VariableSubspaceHelper<eBoundSize> calibratedVariableSubspace() const {
+    FullProjectorMapping fullMapping = fullProjectorMapping();
+    return VariableSubspaceHelper<eBoundSize>(
+        std::span(fullMapping.begin(), fullMapping.begin() + calibratedSize()));
+  }
+
+  template <std::size_t measdim>
+  FixedSubspaceHelper<eBoundSize, measdim> calibratedFixedSubspace() const {
```

Comment:
Not important, but maybe we should stick to the naming of default being fixed and the opposite being called *Effective*?